src/pykeen/trackers/file.py

```diff
@@ -0,0 +1,69 @@
+"""Tracking results in local files."""
+
+import logging
+import pathlib
+from typing import Any, Mapping, Optional, TextIO, Union
+
+from .base import ResultTracker
+
+__all__ = [
+    "CSVResultTracker",
+]
+
+logger = logging.getLogger(__name__)
+
+
+class CSVResultTracker(ResultTracker):
+    """
+    Tracking results to a CSV file.
+
+    Also allows monitoring experiments, e.g. by
+
+    .. code ::
+        tail -f results.txt | grep "hits_at_10"
+    """
+
+    #: The file where the results are written to.
+    file: TextIO
+
+    def __init__(self, path: Union[None, str, pathlib.Path] = None):
```

Comment:
would be good to make the separator a parameter. I think tabs are better than CSV, but sometimes [clown separated values](https://twitter.com/vcsjones/status/1170828523881271298?lang=en) are the only way to go.